dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.cdi3.0-appSecurityClient.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.cdi3.0-appSecurityClient
+visibility=private
+IBM-Process-Types: client, \
+ server
+IBM-Provision-Capability: \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.cdi-3.0))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.appSecurityClient-2.0))"
+-bundles=\
+ com.ibm.ws.cdi.client
```

Comment:
does `com.ibm.ws.cdi.client` bundle need to be transformed?

Reply:
No. It does not depend on any Java EE classes.